src/getting-started/index.md

```diff
@@ -1 +1,3 @@
 # Getting Started
+In this chapter, we will deep dive on how to set up your environment for developing applications with Rust on Espressif devices
```

Comment:
```suggestion
In this chapter, we will explain in detail how to set up your environment for developing applications with Rust on Espressif devices.
```

This feels a bit awkward to me as-is, feel free to ignore the suggestion though. Missing period at the end of the sentence too, though :)